docs/spdx-tag-value-mapping.md

```diff
@@ -0,0 +1,151 @@
+
+## Relevant Fields
+
+The following table contains (1) all fields that are designated by the SPDX specification as 'mandatory', for the SPDX elements that Tern is likely to use; and (2) any additional fields that are optional but likely to be useful for Tern.
+
+Thoughts on mapping these fields to Tern's data model are described in the **Mapping** section below.
+
+Section (§) numbers are references to the relevant portions of [the SPDX specification](https://spdx.org/specifications), version 2.1. Examples should be assumed to be on a single line, though a Markdown formatter might split them across lines.
+
+### Document Creation
+
+The following fields should appear *once*, at the beginning of the SPDX document:
+
+ §  | SPDX Field name | Mandatory? | Brief description    | Example
+----|-----------------|------------|----------------------|---------
+2.1 | SPDX Version    |    Yes     | version of SPDX spec | `SPDXVersion: SPDX-2.1`
+2.2 | Data License    |    Yes     | license for SPDX metadata in the document itself; always `CC0-1.0` | `DataLicense: CC0-1.0`
+2.3 | SPDX Identifier |    Yes     | identifier for the SPDX document itself; always `SPDXRef-DOCUMENT` | `SPDXID: SPDXRef-DOCUMENT`
+2.4 | Document Name   |    Yes     | human-readable name for the SPDX document itself | `DocumentName: Tern report for ACME Dockerfile`
+2.5 | SPDX Document Namespace | Yes| unique absolute URI for the SPDX document itself | `DocumentNamespace: https://example.com/spdxdocs/tern-report-ACME-1.0.1-123456`
+2.7 | License List Version    | No | release version of the SPDX License List being used | `LicenseListVersion: 3.4`
+2.8 | Creator         |    Yes     | one or more people, orgs or tools used to create the SPDX document | `Creator: Tool: tern-0.4.0`
+2.9 | Created         |    Yes     | the time and date when the SPDX document was created (ISO 8601; UTC) | `Created: 2019-03-15T08:25:00Z`
+
+### Package
+
+The following fields should appear once per Package described in the SPDX document. Each Package section must start with the `PackageName` field.
+
+ §  | SPDX Field name  | Mandatory? | Brief description    | Example
+----|------------------|------------|----------------------|---------
+3.1 | Package Name     |    Yes     | full name of package | `PackageName: requests`
+3.2 | Package SPDX Identifier | Yes | SPDX identifier for this package, for reference within this document | `SPDXID: SPDXRef-python-requests-3.1.5`
+3.3 | Package Version  |     No     | the package's version | `PackageVersion: 3.1.5`
+3.7 | Package Download Location | Yes | where the package can be downloaded, or `NOASSERTION` if unknown or choose not to provide | `PackageDownloadLocation: git://github.com:swinslow/tern.git@5cd034e02ebaa4b680a9ffbe1a6d8ea11aef4d84`
+3.8 | Files Analyzed   |    Yes     | *** - see below        | `FilesAnalyzed: false`
```

Comment:
Hmm...Looks like it's time to add a `file` data model.

Reply:
But we can't do it under the `Packages` part as there is no way to know what files belong to what package.